src/views/Home.vue

```diff
@@ -1,18 +1,38 @@
 <template>
   <div class="home">
-    <img alt="Vue logo" src="../assets/logo.png">
-    <HelloWorld msg="Welcome to Your Vue.js App"/>
+    <input type="text" v-model="user" @keyup="getUsers"/>
+    <UsersList :users="users"></UsersList>
   </div>
 </template>
 
 <script>
-// @ is an alias to /src
-import HelloWorld from '@/components/HelloWorld.vue';
+import UsersList from "@/components/UsersList.vue";
+import axios from "axios";
+import VueAxios from "vue-axios";
```

Comment:
This `VueAxios` is not being used here. Do you need it? If not, you can remove it